apps/photo-cleaner/src/index.ts

```diff
@@ -9,14 +9,106 @@
 
 import * as logger from "firebase-functions/logger";
 import { onRequest } from "firebase-functions/v2/https";
+// Some environments may not have up-to-date type declarations for these v2 paths.
+// @ts-ignore
+import { onObjectFinalized } from "firebase-functions/v2/storage";
+// @ts-ignore
+import { onSchedule } from "firebase-functions/v2/scheduler";
+// @ts-ignore
+import * as admin from "firebase-admin";
```

Comment:
nits: こちら、 @ts-ignore 外せますかね
```suggestion
import { onObjectFinalized } from "firebase-functions/v2/storage";
import { onSchedule } from "firebase-functions/v2/scheduler";
Comment viewimport * as admin from "firebase-admin";
```